config/components/crd/bases/kueue.x-k8s.io_workloads.yaml

```diff
@@ -7461,6 +7462,8 @@ spec:
                   - name
                   - spec
                   type: object
+                maxItems: 8
```

Comment:
What happens when you do `kubectl explain workloads`? Do these limits show in the help? if so, we can get rid of the comments of the form "this field must have at most 8 items"

Reply:
```bash
# kubectl explain workloads.spec.podSets
KIND:     Workload
VERSION:  kueue.x-k8s.io/v1alpha2

RESOURCE: podSets <[]Object>

DESCRIPTION:
     podSets is a list of sets of homogeneous pods, each described by a Pod spec
     and a count. There must be at least one element and at most 8. podSets
     cannot be changed.

FIELDS:
   count	<integer> -required-
     count is the number of pods for the spec.

   name	<string> -required-
     name is the PodSet name.

   spec	<Object> -required-
     spec is the Pod spec. If requests are omitted for a container or
     initContainer, they default to the limits if they are explicitly specified
     for the container or initcontainer.
```

Reply:
I take that the answer is: they don't show up, so we should keep the description as-is.